Source/Core/VideoBackends/Vulkan/CMakeLists.txt

```diff
@@ -57,10 +57,10 @@ endif()
 
 # Only include the Vulkan headers when building the Vulkan backend
 target_include_directories(videovulkan
-PRIVATE
-  ${CMAKE_SOURCE_DIR}/Externals/Vulkan-Headers/include
-  ${CMAKE_SOURCE_DIR}/Externals/VulkanMemoryAllocator/include
-  ${CMAKE_SOURCE_DIR}/Externals/libadrenotools/include
+        PRIVATE
+        ${CMAKE_SOURCE_DIR}/Externals/Vulkan-Headers/include
+        ${CMAKE_SOURCE_DIR}/Externals/VulkanMemoryAllocator/include
+        ${CMAKE_SOURCE_DIR}/Externals/libadrenotools/include
```

Comment:
I don't disagree with this formatting change, but since there are no other changes besides formatting I'm not sure it should be in this PR.
Nitpick though, leave it or not. I have no preference, but a graphics reviewer might.